engine/flink/test-utils/src/main/resources/docker/conf.yml

```diff
@@ -2,3 +2,5 @@ high-availability: NONE
 parallelism.default: 1
 taskmanager.memory.jvm-metaspace.size: 512m
 taskmanager.numberOfTaskSlots: <configured_by_flink_entrypoint>
+jobmanager.memory.process.size: 2gb
```

Comment:
Are these numbers not too big?

Reply:
changed it to narrower `jobmanager.memory.flink.size` with 800mb. With 500mb it was blowing up.